includes/hooks.php

```diff
@@ -0,0 +1,108 @@
+<?php
```

Comment:
Instead of having this as a non-class PHP file, can we add this function to the `class-spaces-pages.php` class in the `load_dependencies` function? That way you will also have access to the $settings array which contains the api keys and such.

Reply:
@bAngerman I've refactored this, please take another look and let me know if this is more along the lines of what you'd like to see.